### ISSUE TITLE
Error: Cannot find module '/tmp/.mount_ircclowtOiPO/resources/electron.asar/browser/init.js'

### ISSUE THREAD

**Issue by askb** [irccloud/irccloud-desktop#121]
I started getting this error on Fedora 28, until recently was working without any issues. Is this a known issue?

```
 ./irccloud-desktop-0.7.0-x86_64.AppImage 
module.js:545
    throw err;
    ^

Error: Cannot find module '/tmp/.mount_ircclowtOiPO/resources/electron.asar/browser/init.js'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:473:25)
    at Function.Module.runMain (module.js:683:10)
    at startup (bootstrap_node.js:196:16)
    at bootstrap_node.js:622:3

```

**Comment by jwheare** [comment]
Thanks for the report. Looks like this is the culprit: https://github.com/electron-userland/electron-builder/issues/2958

We'll take a look and see what can be done.

**Comment by askb** [comment]
Is there a workaround available for the issue or any alternatives to using Appimage?

**Comment by jwheare** [comment]
You could try installing the .deb

**Comment by jwheare** [comment]
@askb this should now be fixed in v0.7.1. Could you try updating the app or reinstalling it and let me know?

**Comment by askb** [comment]
@jwheare thanks, works now.
